### PR TITLE
Updated client library page to reflect official php libary

### PR DESCRIPTION
**Pull request by thecodeassassin**

### CODE REVIEW

File: content/docs/v0.9/clients/api.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ This is a list of the client libraries which currently support InfluxDB version
 
 ## PHP
 
+- **Official** [InfluxDB-PHP Github](https://github.com/influxdb/influxdb-php) by [InfluxDB](https://github.com/influxdb)
```
Comment (beckettsean) [inline content/docs/v0.9/clients/api.md]: @pauldix @toddboom we have no other libraries declared "Official". Is there criteria for such? Does the influxdb-php repo meet the informal criteria? Are there others on this page (Java, Ruby, Rails) that we should also label as "Official"

Comment (pauldix) [inline content/docs/v0.9/clients/api.md]: hmmm. The only one I would absolutely call official is the Go library. I guess we should point people at what we think is the best one. For the PHP one I think that's the one if it's the new one that codeassasin has been working on.

Comment (beckettsean) [inline content/docs/v0.9/clients/api.md]: How about "Official" for InfluxDB-maintained libraries, which currently means only the Go library. Then "Recommended" for most favored versions per language? Maybe Recommended implies too much testing/compatibility on our part? Maybe "Preferred" or "Primary" or some neat BttF reference I can't think of right now?

**Comment by beckettsean** [comment]
@thecodeassassin until we have an official certification regime we aren't comfortable recommending or blessing a particular repo over another.

**Comment by beckettsean** [comment]
Added in a direct commit without the "Official" language.

**Comment by thecodeassassin** [comment]
Thanks @beckettsean

**Pull request closed without merging**